### ISSUE TITLE
problems creating chain on OSX

### ISSUE THREAD

**Issue by joshuakarjala** [eris-ltd/eris-cli#325]
Newest version of OSX.
Docker 1.9.0

When I try to create a test chaing I get the following errors.

```
Tragic! Our marmots encountered an error during setupChain for eris_chain_test_chain_1.
They also failed to cleanup after themselves (remove containers) due to another error.
First error =>			Could not import the data container.
```

**Comment by compleatang** [comment]
@joshuakarjala can you dump me the debug output (rerun the command with `--debug`)?

**Comment by joshuakarjala** [comment]
@compleatang using both the darwin binary and the go lang install version

looks like it's trying to install into /home/eris which doesn't exist

```
eris -d chains new test_chain                                                                                                                                    [0:46:31]
Connecting to the Docker Client via =>	tcp://192.168.99.101:2376:/Users/joshua/.docker/machine/machines/eris
Checking Details via Docker-Machine =>	eris
Connecting to the Docker Client via =>	tcp://192.168.99.101:2376:/Users/joshua/.docker/machine/machines/eris
Connecting to the Docker Client via TLS.
	URL =>			tcp://192.168.99.101:2376
	Docker Cert Path =>	/Users/joshua/.docker/machine/machines/eris
Connected over TLS.
Successfully connected to Docker daemon.
Setting IPFS Host
Set ERIS_IPFS_HOST to =>	http://192.168.99.101
Starting Setup for ChnID =>
There are no containers.
Creating Data Container for =>	test_chain
Parsing Containers =>		eris_data_test_chain_1:true
ALL: true
Container Not Found =>		eris_data_test_chain_1
Data Container ID =>		944f54c9af9521e7814062342fb21f4e3659793c00842370581551479276ca31
Chain's Data Contain Built =>	test_chain
Container destination =>	blockchains/test_chain
Local destination =>		/Users/joshua/.eris/data/test_chain/blockchains/test_chain
Copying chain files into the correct location.
	Copying files =>	/Users/joshua/.eris/blockchains/default:/Users/joshua/.eris/data/test_chain/blockchains/test_chain
Copying Files into DataCont =>	/Users/joshua/.eris/data/test_chain/blockchains/test_chain:blockchains/test_chain
Found Data container =>		test_chain:test_chain and 1:1
Importing FROM =>		/Users/joshua/.eris/data/test_chain
Importing TO =>			/home/eris/.eris
Error on setupChain =>		Could not import the data container.
Tried with docker --rm =>	exit status 1
Tried without docker --rm =>	exit status 1
Cleaning up...
Loading Chain Definition =>	test_chain:1
Chain Defaults Set. Image =>	eris/erisdb:0.10.3
Tragic! Our marmots encountered an error during setupChain for eris_chain_test_chain_1.
They also failed to cleanup after themselves (remove containers) due to another error.
First error =>			Could not import the data container.
Tried with docker --rm =>	exit status 1
Tried without docker --rm =>	exit status 1
Cleanup error =>		Unable to load the chain's config for test_chain in /Users/joshua/.eris/blockchains.
Check your known chains with:
eris chains known
```

**Comment by compleatang** [comment]
Josh, am pretty sure this is an OSX thing. It has to do with docker's layers and a hack we had to do to copy files into data containers. We've fixed this on develop (now we can use docker's APIs to abstract that copying hack). Develop branch is basically (0.11) release candidate ready so it is pretty stable. To update use `eris update --branch develop` (this will require go to be installed). In the alternative if it is an option to move to a linux environment via a Vagrant box or remote boxes then all the tests pass. Am still asking around but I just rebuilt current master and pulled the current 0.10.3 containers and wasn't able to recreate from linux.
